bucket/dxc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.7.2212.1",
+    "description": "DirectXShaderCompiler is a collection of tools, libraries and tests for compiling HLSL into DXIL, SPIR-V, and other target languages.",
+    "homepage": "https://github.com/microsoft/DirectXShaderCompiler",
+    "license": "UIUC",
+    "url": "https://github.com/microsoft/DirectXShaderCompiler/releases/download/v1.7.2212.1/dxc_2023_03_01.zip",
+    "hash": "e4e8cb7326ff7e8a791acda6dfb0cb78cc96309098bfdb0ab1e465dc29162422",
+    "architecture": {
+        "64bit": {
+            "env_add_path": "bin/x64"
+        },
+        "32bit": {
+            "env_add_path": "bin/x86"
+        },
+        "arm64": {
+            "env_add_path": "bin/arm64"
+        }
```

Comment:
Does it need env or just shimming?

Reply:
It's just for shimming. I somehow wrote env here, I can change it to `"bin": "bin/x64/dxc.exe"`

Reply:
Ah, I now remember what was the reason. Because there is another dxc executable is shipped by vulkan and it's added into env. 

https://github.com/ScoopInstaller/Main/blob/bb03e28d847f8406b13e349c3cce3ba857f18893/bucket/vulkan.json#L31-L36

This will prioritize the dxc in vulkansdk over the shims (because the path is newly added before scoop's shim folder path). I then wrote env for workaround.